google/resource_container_cluster.go

```diff
@@ -1555,3 +1556,17 @@ func masterAuthClientCertCfgSuppress(k, old, new string, r *schema.ResourceData)
 
 	return strings.HasSuffix(k, ".issue_client_certificate") && old == "" && new == "true"
 }
+
+func podSecurityPolicyCfgSuppress(k, old, new string, r *schema.ResourceData) bool {
+	if strings.HasSuffix(k, "pod_security_policy_config.#") && old == "1" && new == "0" {
```

Comment:
The `HasSuffix` doesn't make much sense if the next step is to `GetOk` an unprefixed `pod_security_policy_config` - should it just be '=='?

Reply:
I wasn't sure if k actually starts with pod_security_policy_config or has resource-specific naming prefix (i.e. google_container_cluster.my_cluster.pod_security_policy_config.#). Since it wasn't the same value necessarily as the keys for r, I thought I'd err on the cautious side but I can change it.